server/api/src/rsGetHostForGet.cpp

```diff
@@ -17,8 +17,13 @@
 #include "irods/rsGetHostForGet.hpp"
 #include "irods/irods_resource_backport.hpp"
 #include "irods/irods_resource_redirect.hpp"
+#include "irods/irods_at_scope_exit.hpp"
 
+#include <nlohmann/json.hpp>
+
+#include <set>
 #include <cstring>
+#include <optional>
```

Comment:
Looks like the changes in this file aren't needed anymore?

Reply:
Taken care of....